// typedoc-plugin-markdown/dist/options/maps.js
"use strict";
/**
 * Maps a given value to the option type.
 *
 * @module
 */
Object.defineProperty(exports, "__esModule", { value: true });
exports.IndexFormat = exports.ReflectionFormat = exports.OutputFileStrategy = void 0;
/**
 * The allowed values of the `--outputFileStrategy` option.
 */
var OutputFileStrategy;
(function (OutputFileStrategy) {
    OutputFileStrategy["Members"] = "members";
    OutputFileStrategy["Modules"] = "modules";
})(OutputFileStrategy || (exports.OutputFileStrategy = OutputFileStrategy = {}));
/**
 * The allowed values for formatting reflections.
 */
var ReflectionFormat;
(function (ReflectionFormat) {
    ReflectionFormat["List"] = "list";
    ReflectionFormat["Table"] = "table";
    ReflectionFormat["HtmlTable"] = "htmlTable";
})(ReflectionFormat || (exports.ReflectionFormat = ReflectionFormat = {}));
/**
 * The allowed values for formatting indexes.
 */
var IndexFormat;
(function (IndexFormat) {
    IndexFormat["List"] = "list";
    IndexFormat["Table"] = "table";
})(IndexFormat || (exports.IndexFormat = IndexFormat = {}));
